packages/workshop-utils/src/epic-api.server.ts

```diff
@@ -266,21 +296,29 @@ async function getEpicProgress({
 		offlineFallbackValue: [],
 		checkValue: EpicProgressSchema,
 		async getFreshValue(context): Promise<z.infer<typeof EpicProgressSchema>> {
-			const response = await fetch(`https://${host}/api/progress`, {
+			const progressUrl = `https://${host}/api/progress`
+			log(`making progress API request to: ${progressUrl}`)
+			
+			const response = await fetch(progressUrl, {
 				headers: {
 					authorization: `Bearer ${authInfo.tokenSet.access_token}`,
 				},
 			}).catch((e) => new Response(getErrorMessage(e), { status: 500 }))
+			
+			log(`progress API response: ${response.status} ${response.statusText}`)
+			
 			if (response.status < 200 || response.status >= 300) {
-				console.error(
-					`Failed to fetch progress from EpicWeb: ${response.status} ${response.statusText}`,
-				)
+				log(`failed to fetch progress from EpicWeb: ${response.status} ${response.statusText}`)
```

Comment:
I think I would like to be able to do:

```ts
log.error(`...`)
log.info(`...`)
// warn, etc.
```

These should prefix with an emoji to make it easier to differentiate. Calling `log` directly should still be the most common and that should not result in an emoji prefix.